niceman/resource/aws_ec2.py

```diff
@@ -297,4 +300,13 @@ def create_key_pair(self, key_name=None):
         # Save the new info to the resource. This is later picked up and
         # saved to the resource inventory file.
         self.key_name = key_name
-        self.key_filename = key_filename
\ No newline at end of file
+        self.key_filename = key_filename
+
+    def login(self):
+        """
+        Log into a container and get the command line
+        """
+        lgr.debug("Opening TTY connection to AWS EC2 instance.")
+        host = self._ec2_instance.public_ip_address
+        ssh = SSHClient(host, private_key=self.key_filename)
+        ssh.interactive_shell(self.user)
```

Comment:
could we also get a simple "SSHHost" resource or smth like that so we could register/login to remote hosts (which do not need startup etc) general?